bundles/org.openhab.binding.boschshc/src/test/java/org/openhab/binding/boschshc/internal/devices/bridge/BoschHttpClientTest.java

```diff
@@ -278,7 +278,7 @@ void sendRequestInvalidSyntaxInResponse()
         ExecutionException e = assertThrows(ExecutionException.class,
                 () -> httpClient.sendRequest(request, SubscribeResult.class, sr -> false, null));
         assertEquals(
-                "Received invalid content in response, expected type org.openhab.binding.boschshc.internal.devices.bridge.dto.SubscribeResult: com.google.gson.stream.MalformedJsonException: Unterminated string at line 1 column 44 path $.@type",
+                "Received invalid content in response, expected type org.openhab.binding.boschshc.internal.devices.bridge.dto.SubscribeResult: com.google.gson.stream.MalformedJsonException: Unterminated string at line 1 column 44 path $.@type\nSee https://github.com/google/gson/blob/main/Troubleshooting.md#malformed-json",
```

Comment:
Looking at this assertion again, I realize that only the first part comes from our openHAB code. We should not test messages from external libraries since this makes our tests unstable. Therefore I would propose to test only that the exception message starts with our message:

```java
assertThat(e.getMessage(), startsWith(
                "Received invalid content in response, expected type org.openhab.binding.boschshc.internal.devices.bridge.dto.SubscribeResult:"));
```

This check uses Hamcrest assertions as recommended in the [unit test guidelines](https://www.openhab.org/docs/developer/tests.html). The corresponding imports are:

```
import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.core.StringStartsWith.startsWith;
```

Reply:
@david-pace done as requested

Reply:
Perfect, thank you very much :+1: